lfs/setup.go

```diff
@@ -26,8 +26,7 @@ var (
 	filters = &Attribute{
 		Section: "filter.lfs",
 		Properties: map[string]string{
```

Comment:
This is just my test setup. Eventually I would keep "clean" and "smudge" because the Git implementation will fall back to these commands if the "driver" fails... this would be a good safety net especially when the new implementation is rolled out for the first time... (although then we won't learn about the errors 😄 ).

BTW: do you think "driver" is an acceptable name? if not, do you have better ideas?